docs/Configuration-Files.md

```diff
@@ -2,48 +2,83 @@
 
 ## Configuration loading
 
-Configuring `reek` via configuration file is by far the most powerful way.
 
-There are 3 ways of passing `reek` a configuration file:
+Configuring `reek` via a configuration file is by far the most powerful way.
```

Comment:
The changes in this file are basically due a copying of https://github.com/troessner/reek#configuration-file into this file.
So yes, it is duplicated. I still think this the right way to go in this special case because:

1.) we want to link to this doc independently 
2.) while maintaining this piece of information in the README for conversion reasons. Or in non-marketing speak: If we move the information from the README to the /docs directory a lot people will jump the ship and not use reek because the README is incomplete and they need to jump over the mental barrier of leaving the README for something that is considered a basic thing.

Alternatively I'd vote for removing /docs/Configuration-Files.md.
WDYT?